packages/amplify-cli/src/telemetry/getConfig.ts

```diff
@@ -0,0 +1,11 @@
+import { Config } from './config';
+import os from 'os';
+import { Context } from '../domain/context';
+import * as path from 'path';
+
+export function getConfig(context: Context): Config {
```

Comment:
As per out discussion the file should be: `$HOME/.amplify/<executable>-configuration.json`.